packages/paste-core/utilities/text/src/index.tsx

```diff
@@ -16,7 +16,6 @@ import {ThemeShape} from '@twilio-paste/theme-tokens';
 
 interface MarginProps {
   m?: keyof ThemeShape['space'];
-  margin?: keyof ThemeShape['space'];
```

Comment:
Curious why you chose to remove just `margin` from the interface?

Reply:
I was getting a typescript error about it... but I am not anymore.  Re-added it